iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift

```diff
@@ -7,15 +7,49 @@
 //
 
 import SwiftUI
+import shared
 
 struct AddMealView: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
+
+	@ObservedObject private(set) var viewModel: ViewModel
+	
+	var body: some View {
+		//TODO: Pass the actual values when UI is complete
+		Button("Add a meal") {
+			let meal = Meal(id: "\(viewModel.sdk.getUUID())",
+											name: "Pizza",
```

Comment:
The alignment seems a bit too far to the right to me.  Could we align them to `id: ...` above?

Reply:
ahh..I think that's just GitHub rending it that way...Looks fine in Xcode  :)